projects/plugins/social/src/class-meta-tags.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * Adds meta tags to pages that need it.
+ *
+ * @package automattic/jetpack-social-plugin
+ */
+
+namespace Automattic\Jetpack\Social;
+
+/**
+ * Adds the meta tags.
+ */
+class Meta_Tags {
+	/**
+	 * This list is copied verbatim from class.jetpack.php
+	 *
+	 * Note: All in One SEO Pack, All in one SEO Pack Pro, WordPress SEO by Yoast, and WordPress SEO Premium by Yoast automatically deactivate
+	 * Jetpack's Open Graph tags via filter when their Social Meta modules are active.
+	 *
+	 * @var array Array of plugin slugs.
+	 */
+	private $open_graph_conflicting_plugins = array(
```

Comment:
Is there another package that this could live in and be shared by us and Jetpack? Perhaps we could put it in the Publicize package?

Reply:
We could, but I intentionally copied it here. I'd like to replace this with a proper implementation of meta tags ASAP, rather than adding the list to a semi-unrelated package, updating Jetpack to use that, then later updating Jetpack again to remove it.

That being said, I'm not opposed if you think we should avoid copying it (after all, [nothing is more permanent than a temporary solution](https://blog.juliobiason.me/books/things-i-learnt/permanent-solution/#:~:text=Depending%20on%20where%20you%20look,fix%20them%2C%20will%20become%20permanent.) 😄).

Reply:
I think that's a good idea actually! Should we do that in a separate PR?

Reply:
Yep, sounds good. We can keep it here for now and extract it after the beta release.